### ISSUE TITLE
Color is not correct

### ISSUE THREAD

**Issue by BlueMatthew** [timothybrooks/hdr-plus#86]
I merged 4 dng photos taken by android camera. and output photo's color seems not correct.

![hdrplus](https://github.com/user-attachments/assets/50a1dd63-b393-4484-8dc4-60a12171e06b)

Below links are original dng photos and output photo:
 http://www.wakinchau.net/raws/out.png
 http://www.wakinchau.net/raws/1.dng
 http://www.wakinchau.net/raws/2.dng
 http://www.wakinchau.net/raws/3.dng
 http://www.wakinchau.net/raws/4.dng

**Comment by Titaniumtown** [comment]
I'll take a look, probably an issue with the recent updates I did.

**Comment by Titaniumtown** [comment]
@BlueMatthew why are the links dng files, but when I download it they're tiffs? idk if this matters but that's kinda weird.

**Comment by BlueMatthew** [comment]
Yes, DNG created by DngCreator in android DOES use TIFF format following the DNG spec.
